website/versioned_docs/version-5.x/server-side-rendering-overview.md

```diff
@@ -8,7 +8,7 @@ sidebar_label: Overview
 
 In the context of single page applications (SPAs), server-side rendering (SSR) refers to dynamic generation of the HTML page that is sent from web server to browser. In a single page application, the server only generates the very first page that the user requests, leaving all subsequent pages to be rendered by the browser.
 
-To accomplish server-side rendering of an SPA, javascript code is executed in NodeJS to generate the initial HTML. In the browser, the same javascript code is executed during a "hydration" process, which attaches event listeners to the HTML. Most popular UI Frameworks (Vue, React, Angular, etc) are capable of executing in both NodeJS and the browser, and offer APIs for both generating the server HTML and hydrating it in the browser. Additionally, there are popular frameworks such as NextJS and Nuxt which simplify the developer experience of server-side rendering.
+To accomplish server-side rendering of a SPA, javascript code is executed in NodeJS to generate the initial HTML. In the browser, the same javascript code is executed during a "hydration" process, which attaches event listeners to the HTML. Most popular UI Frameworks (Vue, React, Angular, etc) are capable of executing in both NodeJS and the browser, and offer APIs for both generating the server HTML and hydrating it in the browser. Additionally, there are popular frameworks such as NextJS and Nuxt which simplify the developer experience of server-side rendering.
```

Comment:
I believe `an` is actually correct here, as "SPA" is an acronym and you actually say the letter "ess" out loud. See also https://www.grammar.com/a-vs-an-when-to-use

Reply:
Wow haha learn something new every day! Reverted this, thank you

Reply:
Though maybe enough people say “spa” instead of S.P.A. nowadays… 🤷